docs/api/adbc.md

```diff
@@ -123,3 +130,119 @@ StatementSetOption(&adbc_statement, ADBC_INGEST_OPTION_TARGET_TABLE, "AnswerToEv
 StatementBindStream(&adbc_statement, &arrow_stream, &adbc_error);
 StatementExecuteQuery(&adbc_statement, nullptr, nullptr, &adbc_error);
 ```
+
+### Python
+
+The first thing to do is to use `pip` and install the ADBC Driver manager.
+
+```shell
+pip install adbc_driver_manager
+```
+
+You will also need the `pyarrow` package to directly access Apache Arrow
+formatted result sets.
+```shell
+pip install pyarrow
+```
+
+The full documentation for the `adbc_driver_manager` package can be found
+[here](https://arrow.apache.org/adbc/0.5.1/python/api/adbc_driver_manager.html).
+
+As with C++, we need to provide initialization options for an `AdbcDatabase` object. 
```

Comment:
I don't know we want to document this. One, this will all change when we have PyCapsules. Two, it's really confusing compared to using the DB-API interface below.

Reply:
So, only document the DBAPI interface?

Reply:
I think that's preferable.

Reply:
Okay, i'll update this

Reply:
how's the current version @lidavidm ?

Reply:
LGTM, thanks!